packages/block-library/src/template-part/index.js

```diff
@@ -14,5 +14,8 @@ export { metadata, name };
 
 export const settings = {
 	title: __( 'Template Part' ),
+	supports: {
+		html: false,
```

Comment:
This one already has it in `block.json`, we should remove it there.

Reply:
Oh right. I removed it.